lib/modules/manager/helmv3/artifacts.ts

```diff
@@ -83,20 +72,20 @@ async function helmCommands(
   // add helm repos if an alias or credentials for the url are defined
   classicRepositories.forEach((value) => {
     const { username, password } = value.hostRule;
-    const parameters = [...helmConfigParameters];
+    const parameters = [`${value.repository}`];
     const isPrivateRepo = username && password;
     if (isPrivateRepo) {
       parameters.push(`--username ${quote(username)}`);
       parameters.push(`--password ${quote(password)}`);
     }
 
     cmd.push(
-      `helm repo add ${value.name} ${parameters.join(' ')} ${value.repository}`
+      `helm repo add ${value.name} ${parameters.join(' ')}`
```

Comment:
```suggestion
      `helm repo add ${value.name} ${value.repository}`
```

Reply:
this is necessary because of if isPrivateRepo is true, we need to add username/password parameter.
```
if (isPrivateRepo) {
  parameters.push(`--username ${quote(username)}`);
  parameters.push(`--password ${quote(password)}`);
}
```